storage/flush.go

```diff
@@ -100,8 +103,14 @@ func (m *flushManager) Flush(curr time.Time) error {
 	m.setState(flushManagerFlushInProgress)
 	for _, ns := range namespaces {
 		// Flush first because we will only snapshot if there are no outstanding flushes
-		flushTimes := m.namespaceFlushTimes(ns, curr)
-		multiErr = multiErr.Add(m.flushNamespaceWithTimes(ns, flushTimes, flush))
+		flushTimes := m.namespaceFlushTimes(ns, tickStart)
+		shardBootstrapTimes, ok := dbBootstrapStates.namespaceBootstrapStates[ns.ID().String()]
+		if !ok {
```

Comment:
feels weird to see this return when you have the mutlierr, could you restructure to the following:

```
namespaces, err := m.database.GetOwnedNamespaces()
if err != nil {		
    return err
}		 
for _, ns := range namespaces {
  if _, ok := dbBootstrapStates[ns.ID().String()]; ok {
    return fmt.Errorf(...)
  }
}

// current for _, ns := namespaces {
  ... 
}
```